.github/workflows/jenkins-security-scan.yml

```diff
@@ -0,0 +1,21 @@
+name: Jenkins Security Scan
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [ opened, synchronize, reopened ]
```

Comment:
If i'm reading the [documentation](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target) right, this is redundant as these are the default activity types.

Reply:
True, however it is not the worst idea to have them here anyways, as some maintainers may want to edit them. In addition they are also part of the archetype / template and derivations will likely cause headache later on. 

What do you think? I’d go for leaving them be.

Reply:
The second reason is compelling.  The first not so much - by that logic, we should include the entire set of yaml defaults in every copy.